integration/requirements-frozen.txt

```diff
@@ -0,0 +1,110 @@
+apipkg==1.5
+appdirs==1.4.4
+arrow==0.15.7
+asn1crypto==0.24.0
+atomicwrites==1.4.0
+attrs==17.4.0
+Automat==0.6.0
+backcall==0.2.0
+beakerx==0.16.1
+bleach==3.1.5
+blessed==1.17.8
+certifi==2018.1.18
+cffi==1.14.0
+chardet==3.0.4
+click==6.7
+colorama==0.3.7
+configobj==5.0.6
+constantly==15.1.0
+cook-client==3.0.0
+cryptography==2.1.4
+decorator==4.4.2
+defusedxml==0.6.0
+distlib==0.3.0
+distro-info===0.18ubuntu0.18.04.1
+entrypoints==0.3
+execnet==1.7.1
+filelock==3.0.12
+httplib2==0.9.2
+humanfriendly==8.2
+hyperlink==17.3.1
+idna==2.6
+importlib-metadata==1.6.1
+importlib-resources==2.0.1
+incremental==16.10.1
+ipykernel==5.3.2
+ipython==7.16.1
+ipython-genutils==0.2.0
+ipywidgets==7.5.1
+jedi==0.17.1
+Jinja2==2.11.2
+jsonschema==3.2.0
+jupyter-client==5.2.3
+jupyter-core==4.6.3
+language-selector==0.1
+MarkupSafe==1.1.1
+mistune==0.8.4
+more-itertools==8.4.0
+nbconvert==5.6.0
+nbformat==4.4.0
+netifaces==0.10.4
+notebook==6.0.3
+numpy==1.15.3
+packaging==20.4
+PAM==0.4.2
+pandas==1.0.5
+pandocfilters==1.4.2
+parso==0.7.0
+pexpect==4.8.0
+pickleshare==0.7.5
+pluggy==0.13.1
+prometheus-client==0.8.0
+prompt-toolkit==3.0.5
+ptyprocess==0.6.0
+py==1.9.0
+pyasn1==0.4.2
+pyasn1-modules==0.2.1
+pycparser==2.20
+pygit2==0.28.2
+Pygments==2.6.1
+pygobject==3.26.1
+pyOpenSSL==17.5.0
+pyparsing==2.4.7
+pyrsistent==0.16.0
+pyserial==3.4
+pytest==5.2.0
+pytest-forked==1.2.0
+pytest-timeout==1.3.3
+pytest-xdist==1.30.0
+python-apt==1.6.4
+python-dateutil==2.8.1
+python-debian==0.1.32
+pytz==2020.1
+PyYAML==3.12
+pyzmq==19.0.1
+requests==2.20.0
+requests-unixsocket==0.1.5
+retrying==1.3.3
+Send2Trash==1.5.0
+service-identity==16.0.0
+six==1.11.0
+ssh-import-id==5.7
+systemd-python==234
+tabulate==0.8.7
+tenacity==6.2.0
+terminado==0.8.3
+testpath==0.4.4
+tornado==5.1.1
+traitlets==4.3.3
+Twisted==17.9.0
+tzlocal==2.1
+ufw==0.36
+urllib3==1.22
+virtualenv==20.0.25
+wcwidth==0.2.5
+webencodings==0.5.1
+widgetsnbextension==3.5.1
+zipp==3.1.0
+zope.interface==4.3.2
+file:../cli#egg=cook_client
+pip==9.0.1; python_version >= '3.6'
```

Comment:
Should we use this file in places where we currently use `requirements.txt`? e.g. the Vagrantfile, Dockerfiles, etc.?

Reply:
updated